Updating docs for changes with Operator 0.19.5 (#592)
minio/docs: source/operations/install-deploy-manage/minio-operator-console.rst

deployed on the cluster, whether provisioned through the Operator Console 
or through the :ref:`MinIO Kubernetes Plugin <minio-k8s-deploy-minio-tenant-commandline>`.

Select a listed tenant to open an in-browser view of that tenant's MinIO Console. 
You can use this view to directly manage, modify, expand, upgrade, and delete the tenant through the Operator UI.

Tenant Registration
-------------------

.. versionchanged:: 0.19.5

>    You can register the the Operator's tenants with an existing |SUBNET| subscription within the Operator Console.

Reviewer: ```suggestion
   You can register your MinIO tenants with your |SUBNET| account using the Operator Console.
```

Author: 👍

.. image:: /images/k8s/operator-console-register.png
   :align: center
   :width: 70%
   :class: no-scaled-link
   :alt: MinIO Operator Console Register Screen

#. Select the :guilabel:`Register` tab
#. Enter the :guilabel:`API Key` if you already have it
   
   or
   

>    Select :guilabel:`Get from SUBNET` to obtain an API key

Author: Minor tweaks, but merged.

   
   In the window the appears, enter the username and password you use to sign in to SUBNET.
   The Operator Console retrieves an API key and populates it to the :guilabel:`API Key` field.

#. Select :guilabel:`Register`

Review Your MinIO License
-------------------------

View the license you are using for your MinIO deployments.
MinIO is provided as Open Source Software under the terms of the AGPLv3 license.
Subscribers to |SUBNET| use MinIO under a commercial license.

> To review which license you are using and the features available through different license options, select the :guilabel:`License` tab.

Author: Done
